storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -491,7 +493,7 @@
           EVENTLOGGER-STREAM-ID
           [component-id message-id (System/currentTimeMillis) values]))))
 
-(defmethod mk-threads :spout [executor-data task-datas initial-credentials]
+(defmethod mk-threads :spout [executor-data task-datas initial-credentials throttle-on]
```

Comment:
Yes, this change will effectively disable backpressure. 

@roshannaik 
We still need to look up that value in spout async loop. 
One sketching idea for workaround is that we may not need to look up that value for each iteration, which is based on precondition that it may also not hurt when spout notices about throttle slightly later.

Btw, I guess when nextTuple() takes several time (say, about to 1 ms, not speed of light but still not slow), looking up that value may be not hurt overall performance. When we don't have data for emit, we sleep 1ms in loop for default configuration.

```
topology.spout.wait.strategy: "org.apache.storm.spout.SleepSpoutWaitStrategy"
topology.sleep.spout.wait.strategy.time.ms: 1
```